core/src/toga/app.py

```diff
@@ -651,6 +653,11 @@ def cleanup(app, should_exit):
 
         self._on_exit = wrapped_handler(self, handler, cleanup=cleanup)
 
+    @property
+    def loop(self) -> asyncio.AbstractEventLoop:
+        """The event loop of the app's main thread (read-only)."""
+        return self._impl.loop
```

Comment:
This will be useful to support common patterns of working with background threads, such as shown [here](https://stackoverflow.com/a/77350586).

Reply:
I've contemplated adding something similar in the past. My only hestitation has been that [`asyncio.get_running_loop()` exists](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_running_loop), so having an app-specific version is *technically* redundant - but I guess it doesn't hurt to have a simple alias that is clearly tied to the app given how important the event loop is to app operation.